test/Altinn.Notifications.IntegrationTests/Utils/PostgreUtil.cs

```diff
@@ -71,6 +71,25 @@ public static async Task<NotificationOrder> PopulateDBWithOrderAndEmailNotificat
         return o;
     }
 
+    public static async Task<(NotificationOrder Order, SmsNotification SmsNotification)> PopulateDBWithOrderAndSmsNotification(string? sendersReference = null)
+    {
+        (NotificationOrder order, SmsNotification smsNotification) = TestdataUtil.GetOrderAndSmsNotification();
+        var serviceList = ServiceUtil.GetServices(new List<Type>() { typeof(IOrderRepository), typeof(ISmsNotificationRepository) });
```

Comment:
Same here 🙂 I guess explicitly declaring types would be better. 🙂

Reply:
Same here as well. May be it isn't needed to declare type explicitly here if this is test case! 🤔